_analyzers/token-filters/elision.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Elision
+parent: Token filters
+nav_order: 130
+---
+
+# Elision token filter
+
+The `elision` token filter is used to remove elided characters from words in certain languages. Elision typically occurs in languages such as French, where words are often contracted and combined with the following word, typically by omitting a vowel and replacing it with an apostrophe. 
+
+The `elision` token filter already comes preconfigured in the following [language analyzers]({{site.url}}{{site.baseurl}}/analyzers/language-analyzers/): `catalan`, `french`, `irish`, and `italian`.
+{: .note}
+
+## Parameters
+
+The custom `elision` token filter can be configured with the following parameters.
+
+Parameter | Required/Optional | Data type | Description
+:--- | :--- | :--- | :--- 
+`articles` | Required if `articles_path` is not configured | Array of strings | Defines which articles or short words should be removed when they appear as part of an elision.
+`articles_path` | Required if `articles` is not configured | String | Specifies the path to a custom list of articles that should be removed during the analysis process. 
+`articles_case` | Optional | Boolean | Specifies whether the filter is case sensitive when matching elisions. Default is `false`.
```

Comment:
Can the filter itself be case sensitive? Or do we mean something like "Specifies whether the filter considers/applies case sensitivity when matching elisions"?

Reply:
Technically,  yes, the filter applies case sensitivity, but I think it's understood as is 😄